pages/common/ng.md

```diff
@@ -0,0 +1,27 @@
+# ng
+
+> Angular Command Line Interface (CLI) for creating and managing Angular applications
+
+- Check the version of your current Angular Installation
```

Comment:
Personally, I would not. Since I would know which version I installed during installation. I would maybe check the version from time to time if I forget. But if you think that this is important to be the first command, I am okay with it.

Reply:
No, I see your point and think you're right. Will put it at the end.